aidies-project/client/src/components/SignupPanel.js

```diff
@@ -27,72 +67,172 @@ const SignupPanel = (props) => {
-                    value={lastName}
-                    onChange={(e) => {
-                        setlastName(e.target.value);
-                    }} />
-                <input type="text"
-                    placeholder="Email"
-                    value={email}
-                    onChange={(e) => {
-                        setEmail(e.target.value);
-                    }} />
-                <input type="password"
-                    placeholder="Password"
-                    value={password}
-                    onChange={(e) => {
-                        setPassword(e.target.value);
-                    }} />
-                <input type="password"
-                    placeholder="Confirm Password"
-                    value={confirmPassword}
-                    onChange={(e) => {
-                        setConfirmPassword(e.target.value);
-                    }} />
-                <textarea 
-                    placeholder="Enter a bio"
-                    value={bio}
-                    onChange={(e) => {
-                        setBio(e.target.value);
-                    }} />
-                <input type="url"
-                    placeholder="Website"
-                    value={website}
-                    onChange={(e) => {
-                        setWebsite(e.target.value);
-                    }} />
-                <input type="text"
-                    placeholder="Location (ex. Atlanta, GA)"
-                    value={location}
-                    onChange={(e) => {
-                        setLocation(e.target.value);
-                    }} />
-                <input type="url"
-                    placeholder="Instagram"
-                    value={instagram}
-                    onChange={(e) => {
-                        setInstagram(e.target.value);
-                    }} />
-                <input type="url"
-                    placeholder="Facebook"
-                    value={facebook}
-                    onChange={(e) => {
-                        setFacebook(e.target.value);
-                    }} />
-                <button type="submit">Sign Up</button>
-            </form>
-        </main>
+        return (
+    <Container component="main" maxWidth="xs">
+      <CssBaseline />
+      <div className={classes.paper}>
+        <Typography component="h1" variant="h5">
+                        Sign up
+        </Typography>
+        <form className={classes.form} noValidate onSubmit={handleSubmit}>
+          <Grid container spacing={2}>
+            <Grid item xs={12} sm={6}>
+              <TextField
+                autoComplete="fname"
+                name="firstName"
+                variant="outlined"
+                required
+                fullWidth
+                id="firstName"
+                label="First Name"
+                autoFocus
+                value={firstName}
+                    onChange={(e) => setFirstName(e.target.value)}
```

Comment:
What made you decide to indent this `onChange` prop?

Reply:
vs code being vs code, will correct this.